app/react/components/Tip/SliderTooltip/SliderTooltip.tsx

```diff
@@ -0,0 +1,44 @@
+import Tippy from '@tippyjs/react';
+import clsx from 'clsx';
+import _ from 'lodash';
+
+import 'tippy.js/dist/tippy.css';
+
+import styles from './SliderTooltip.module.css';
+
+export interface Props {
+  value: string;
+  child: React.ReactElement;
+}
+
+export function SliderTooltip({ value, child }: Props) {
+  const id = _.uniqueId('tooltip-container-');
```

Comment:
Is this id required as a classname?

Reply:
Nope, it was with rc-tooltip, but not with tippy good catch